From amit.pundir@linaro.org  Mon Jan  8 16:33:05 2024
From: Amit Pundir <amit.pundir@linaro.org>
Date: Sun,  7 Jan 2024 21:27:02 +0530
Subject: Revert "interconnect: qcom: sm8250: Enable sync_state"
To: Greg KH <gregkh@linuxfoundation.org>, Sasha Levin <sashal@kernel.org>, Georgi Djakov <djakov@kernel.org>, Konrad Dybcio <konrad.dybcio@linaro.org>
Cc: Stable <stable@vger.kernel.org>
Message-ID: <20240107155702.3395873-1-amit.pundir@linaro.org>

From: Amit Pundir <amit.pundir@linaro.org>

This reverts commit 3637f6bdfe2ccd53c493836b6e43c9a73e4513b3 which is
commit bfc7db1cb94ad664546d70212699f8cc6c539e8c upstream.

This resulted in boot regression on RB5 (sm8250), causing the device
to hard crash into USB crash dump mode everytime.

Signed-off-by: Amit Pundir <amit.pundir@linaro.org>
Link: https://lkft.validation.linaro.org/scheduler/job/7151629#L4239
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/interconnect/qcom/sm8250.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/interconnect/qcom/sm8250.c
+++ b/drivers/interconnect/qcom/sm8250.c
@@ -551,7 +551,6 @@ static struct platform_driver qnoc_drive
 	.driver = {
 		.name = "qnoc-sm8250",
 		.of_match_table = qnoc_of_match,
-		.sync_state = icc_sync_state,
 	},
 };
 module_platform_driver(qnoc_driver);
